Wikipedia/Localizations/en.lproj/Localizable.strings

```diff
@@ -547,6 +547,7 @@
 "explore-randomizer" = "Randomizer";
 
 "find-in-page-no-matches" = "0 matches";
+"find-in-page-number-matches" = "%lu / %lu";
```

Comment:
Since we use translatewiki.net for translations, we can't use standard string tokens. `%lu / %lu` should be `$1 / $2`